lib/network/modules/components/physics/BarnesHutSolver.js

```diff
@@ -62,17 +64,27 @@ class BarnesHutSolver {
         node = nodes[nodeIndices[i]];
         if (node.options.mass > 0) {
           // starting with root is irrelevant, it never passes the BarnesHutSolver condition
-          this._getForceContribution(barnesHutTree.root.children.NW, node);
-          this._getForceContribution(barnesHutTree.root.children.NE, node);
-          this._getForceContribution(barnesHutTree.root.children.SW, node);
-          this._getForceContribution(barnesHutTree.root.children.SE, node);
+          this._getForceContributions(barnesHutTree.root, node);
         }
       }
     }
   }
 
 
   /**
+   * @param {Object} parentBranch
+   * @param {Node} node
+   * @private
+   */
+  _getForceContributions(parentBranch, node) {
```

Comment:
A function beginning with `_get` that has no return value...
Now I know this isn't your fault, as you're being consistent with `_getForceContribution`, but this seems wrong.

Reply:
OK. Well, there is a force contribution to be gotten, only it's not returned but added to param `parentBranch`. I can't think of any better names right now.